src/Microsoft.ML.Recommender/Microsoft.ML.Recommender.csproj

```diff
@@ -0,0 +1,14 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <IncludeInPackage>Microsoft.ML.MatrixFactorizationNative</IncludeInPackage>
```

Comment:
Why are we naming the NuGet package `Microsoft.ML.MatrixFactorizationNative`?  Why have `Native` in the name, especially since it contains more than just the "native" assembly?  I think it should just be `Microsoft.ML.MatrixFactorization`.

Reply:
Ok. Let me remove `Native`.

Reply:
Yes, the C/C++ libraries have `Native` appended to their names, but our NuGet packages aren't named with a `Native` suffix: https://www.nuget.org/packages?q=Microsoft.ML.

I'm strictly talking about the NuGet package name here - which is what the end users have to type/find/etc.

Reply:
Do you know where to control the name of Nuget? Do I only need to change 
```
<NativePackageAsset Include="$(NativeAssetsBuiltPath)\$(NativeLibPrefix)MatrixFactorizationNative$(NativeLibExtension)"                                                                                
RelativePath="Microsoft.ML.MatrixFactorizationNative\runtimes\$(PackageRid)\native" />
```
to
```
<NativePackageAsset Include="$(NativeAssetsBuiltPath)\$(NativeLibPrefix)MatrixFactorizationNative$(NativeLibExtension)"                                                                                
RelativePath="Microsoft.ML.MatrixFactorization\runtimes\$(PackageRid)\native" />
```
and the package reference name in all `<IncludeInPackage>Microsoft.ML.MatrixFactorizationNative</IncludeInPackage>`?

Reply:
I hope [this commit](https://github.com/dotnet/machinelearning/pull/1263/commits/586713620a224400adf11d113addbc809df24b34) can resolve this comment. Thanks.